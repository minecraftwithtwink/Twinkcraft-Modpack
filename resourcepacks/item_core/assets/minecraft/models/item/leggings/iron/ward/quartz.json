{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/leggings/base/iron/ward",
        "layer1": "minecraft:trims/items/leggings_trim/ward_quartz"
    }
}