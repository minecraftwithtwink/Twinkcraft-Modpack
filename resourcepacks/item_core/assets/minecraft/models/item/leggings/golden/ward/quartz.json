{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/leggings/base/golden/ward",
        "layer1": "minecraft:trims/items/leggings_trim/ward_quartz"
    }
}